testsuite/integration-arquillian/tests/pom.xml

```
@@ -1414,6 +1423,80 @@
             </build>
         </profile>
 
+        <profile>
+            <id>hot-rod-store</id>
```

```suggestion
            <id>map-storage-hot-rod</id>
```

Usage of this profile should be documented in HOW-TO-RUN.md